chainer/functions/array/copy.py

```diff
@@ -83,4 +59,6 @@ def copy(x, dst):
         -1
 
     """
-    return Copy(dst)(x)
+    assert isinstance(dst, int)
```

Comment:
Shouldn't it accept `Device` as `dst`?

Reply:
Maybe, only accept `int`.
https://github.com/chainer/chainer/pull/3419/files#diff-1b7bc2d145846eadbc5d0f8a8fd90c20R22

Reply:
Is there any reason?
IMO, it would be more convenient for user if it accepts `Device` as well.

Reply:
This is my mistake.
I added test. and I removed this assert.